API.md

```diff
@@ -602,6 +602,54 @@ Get a monitoree via an id, e.g.:
     {
         "url": "http://saraalert.org/StructureDefinition/full-assigned-jurisdiction-path",
         "valueString": "USA, State 1"
+    },
+    {
+      "url": "http://saraalert.org/StructureDefinition/preferred-contact-time",
+      "valueString": "Morning"
+    },
+    {
+      "url": "http://saraalert.org/StructureDefinition/monitoring-plan",
+      "valueString": "Daily active monitoring"
+    },
+    {
+        "url": "http://saraalert.org/StructureDefinition/assigned-user",
+        "valueString": "9999"
```

Comment:
As of your most recent changes, this file is now out of date with the type of this extension, and the names of the date extensions that have changed.

Reply:
Good call, my bad. Updated the documentation here https://github.com/SaraAlert/SaraAlert/pull/521/commits/2f9fa7deaeb2979b97870fb2540e40d02efcac87.